tests/Element/AbstractElement.php

```diff
@@ -8,7 +8,7 @@
 use PHPUnit\Framework\MockObject\MockObject;
 use PHPUnit\Framework\TestCase;
 
-abstract class ElementTest extends TestCase
+abstract class AbstractElement extends TestCase
```

Comment:
What was the purpose of this class renaming?

Reply:
Same as below